<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
	color: #DEDEDE;
    padding: 2px;
	border: 0;
    background: black;
   	opacity: 200;
}

QPushButton {
	border: 0;
	background: #444;
	padding: 8px;
}

QPushButton:hover {
	background: #555;
}

QListWidget {
	background: #404040;
	color: #CECECE;
}

QListWidget {
	padding-left: 16px;
	padding-top: 24px;
}

QListWidget::item {
	padding: 6px;
}

QListView::item:selected {
    border: 1px solid #3498db;
	background: #3498db;
	color: #CECECE;
}

QTabWidget::tab-bar {
    /*min-width:  48px; */
}

QTabBar::close-button {
   /*  image: url(:icons/icons/delete_smol.png); */
    /*subcontrol-position: left;*/
}

QTabWidget::pane {
	margin: -1px;
	/*border-top: 1px solid #1e1e1e;*/
}

QTabBar::tab:selected {
	background-color: #404040;
	/*border-bottom: 1px solid #303030;*/
}

QTabBar::tab {
	background-color: #4d4d4d;
	min-width: 152px;
    min-height: 32px;
    padding-bottom: 1px;
   /* border-top: 1px solid #1e1e1e; */
	/*border-top: 1px solid #1e1e1e; */
 	/*border-right: 1px solid #1e1e1e;*/
	/*border-bottom: 1px solid #1e1e1e;*/
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/app/images/sidepanelimage.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newProject">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>New Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openProject">
        <property name="text">
         <string>Open Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="recent">
      <attribute name="title">
       <string>Recent Projects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="demos">
      <attribute name="title">
       <string>Sample Projects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QListWidget" name="demoList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
